lib/standalone/index.ts

```diff
@@ -43,7 +47,15 @@ function standaloneCode(
       const v = getValidateFunc(schemas[name] as T)
       if (v) {
         const vCode = validateCode(usedValues, v.source)
-        code = _`${code}${_n}exports${getProperty(name)} = ${v.source?.validateName};${_n}${vCode}`
+        if (ajv.opts.code.exportEsm) {
```

Comment:
I am tempted to refactor common parts out

Reply:
Okay will do for this IF block or did you have something bigger in mind?

Reply:
No, only this if